src/database/schema/ChatSchema.ts

```diff
@@ -175,6 +174,16 @@ class ChatSchema implements BaseDbSchema {
     return Promise.resolve(this.fromDatabaseObject(rows.raw()[0]));
   }
 
+  static async isExists(db: SQLiteDatabase, id: string): Promise<ChatSchema> {
+    const selectQuery = `SELECT A.id
+      FROM ${ChatSchema.getTableName()} A
+      WHERE A.id = ?;`;
+
+    const [{rows}] = await db.executeSql(selectQuery, [id]);
+    if (rows.length === 0) return Promise.resolve(null);
+    return Promise.resolve(this.fromDatabaseObject(rows.raw()[0]));
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isExists` function is declared to return a Promise of `ChatSchema`, but in line 183, it returns `null` when no rows are found. This could potentially cause type errors. It would be better to adjust the function's return type to `Promise<ChatSchema | null>` to accurately reflect what it can return.
```diff
- static async isExists(db: SQLiteDatabase, id: string): Promise<ChatSchema> {
+ static async isExists(db: SQLiteDatabase, id: string): Promise<ChatSchema | null> {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->